EventFilter/EcalRawToDigi/plugins/UnpackGPU.cu

```diff
@@ -0,0 +1,340 @@
+#include "EventFilter/EcalRawToDigi/interface/ElectronicsIdGPU.h"
+
+#include "UnpackGPU.h"
+
+namespace ecal {
+  namespace raw {
+
+    __forceinline__ __device__ void print_raw_buffer(uint8_t const* const buffer,
+                                                     uint32_t const nbytes,
+                                                     uint32_t const nbytes_per_row = 20) {
+      for (uint32_t i = 0; i < nbytes; i++) {
+        if (i % nbytes_per_row == 0 && i > 0)
+          printf("\n");
+        printf("%02X ", buffer[i]);
+      }
+    }
+
+    __forceinline__ __device__ void print_first3bits(uint64_t const* buffer, uint32_t size) {
```

Comment:
as above, put them in a central place

Reply:
printing 3 specific bits out of an 8-byte word is defnitely not something of central interest